setup.py

```diff
@@ -77,9 +75,9 @@ def get_long_description():
 
   package_data={
     'pymtl3': [
-      'passes/backends/verilog/import_/verilator_wrapper.c.template',
-      'passes/backends/verilog/import_/verilator_wrapper.py.template',
-      'passes/backends/verilog/tbgen/verilog_tbgen.v.template',
+      # 'passes/backends/verilog/import_/verilator_wrapper.c.template',
```

Comment:
Now that these templates are just importable python objects we can get rid of `package_data` here...